### ISSUE TITLE
Nipple distribution to lesions

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10083]
**Preferred term label:**
Nipple distribution to lesions

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Mammary Paget disease

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000055
- label: Nipple localization
- parents: [Localized (HP:0012838)](https://hpo.jax.org/app/browse/term/HP:0012838)
- pmid: 
- def: Applies to a sign or symptom that is distributed around the nipples or located in the nipples.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10083
- synonyms:

**Comment by pnrobinson** [comment]
@MickeySegal 
corrected ID -  HP:0040402
